### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by janbrennen**
very

### CODE REVIEW

**Comment by janbrennen** [comment]
ps you should make blåhaj the official mascot i will make a different PR for you to close wontfix later for this

File: README.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 ## yaxpeax-x86
 
-x86 decoders implemented as part of the yaxpeax project.
+x86 decoders implemented as part of the yaxpeax project it is a mazing projec.
```
Comment (hdhoang) [inline README.md:3]: I regret the spam.

**Review by hdhoang — CHANGES_REQUESTED** [review]
please tag ”invalid”

**Comment by iximeow** [comment]
yaxpeax is a mazing projec

File: README.md (hunk 1 above)
Comment (janbrennen) [reply README.md:3]: i regret being alive

Comment (iximeow) [reply README.md:3]: you're alive forever in the commit history

Comment (janbrennen) [reply README.md:3]: oh no

**Pull request merged**